src/MonticelloRemoteRepositories/MCFtpRepository.class.st

```diff
@@ -108,12 +108,12 @@ MCFtpRepository >> password: passwordString [
 { #category : #required }
 MCFtpRepository >> readStreamForFileNamed: aString do: aBlock [
 	
-	^ self clientDo:
-		[:client | | stream |
+	| bytes |
+	
+	^ self clientDo: [:client | 
 		client binary.
-		stream := RWBinaryOrTextStream on: String new.
-		stream nextPutAll: (client getFileNamed: aString).
-		aBlock value: stream reset]
+		bytes := ByteArray streamContents: [ :stream | stream nextPutAll: (client getFileNamed: aString) asByteArray ].
```

Comment:
@tesonep 
Can we replace this line  `bytes := (client getFileNamed: aString) asByteArray ` ?

Reply:
Thanks!